samples/attestations/verify_attestation_chains.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This application verifies HSM certificate chains.
+
+For more information, visit https://cloud.google.com/kms/docs/attest-key.
+"""
+
+# [START kms_verify_chains]
+import argparse
+import gzip
+import io
+import zipfile
+
+from cryptography import exceptions
+from cryptography import x509
+from cryptography.hazmat import backends
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import padding
+import pem
+import requests
+
+ATTESTATION_SIGNATURE_LEN = 256
+
+MANUFACTURER_CERT_URL = 'https://www.marvell.com/content/dam/marvell/en/public-collateral/security-solutions/liquid_security_certificate.zip'
```

Comment:
What is this file? Why do we need to use a file from 3rd party?

Reply:
It's the HSM manufacturer root certificate. We need it to verify the attestation through the manufacturer certificate chain.

Reply:
Gotcha